src/OpenTelemetry.Extensions.Enrichment/README.md

```diff
@@ -1,5 +1,10 @@
 # OpenTelemetry .NET SDK telemetry enrichment framework
 
+| Status        |           |
+| ------------- |-----------|
+| Stability     |  [Development](..\..\Readme.md#development)|
+| Code Owners   |  |
```

Comment:
lets do a follow up to see if there are owners, else start removing components.

Reply:
Yes!